### DISCUSSION TITLE
Are there webshare.io altenatives without a CC?

### DISCUSSION THREAD

**Discussion by strangerobj** [KevinBytesTheDust/CVAmp#197] [Q&A] [answered]
Sorry for the dumb question but I went trough the internet for looking some proxy servers but they are all so expensive compared to webshare. I really would buy from webshare but I dont have a credit card. 

Any recommendations for buying proxies somewhere? Or is there a way to buy proxies from webshare without a credit card? I would appreciated any answer :)

Is there also a way to contact you Mr. jlplenio?

**Comment by jlplenio** [comment, marked as answer]
Hey @strangerobj. you are right. I haven't seen any cheaper prices than [webshare.io](https://www.webshare.io/?referral_code=w6nfvip4qp3g). And I've been using their services for over a year with no problems, including a very attentive and helpful customer support. Unfortunately, I can't tell you any trustworthy and cost-effective alternatives. Maybe others can?

**Reply by strangerobj** [reply]
Can I maybe have your Discord? I have some question about it, if its ok for you. You can also just add me, my discord is menvayne_76205

**Reply by jlplenio** [reply]
Hey @strangerobj, I do not use Discord. You can shoot me an email, it's on my profile.
